tests/architecture_prototype/conftest.py

```diff
@@ -0,0 +1,45 @@
+import shutil
+from pathlib import Path
+from typing import Dict, Text, Any
+
+import pytest
+
+from rasa.architecture_prototype import graph
+from tests.architecture_prototype.graph_schema import (
+    full_model_train_graph_schema,
+    predict_graph_schema,
+)
+
+MODEL_DIR = "model"
+
+
+def clean_directory():
+    # clean up before testing persistence
+    cache_dir = Path(MODEL_DIR)
+    shutil.rmtree(cache_dir, ignore_errors=True)
+    cache_dir.mkdir()
+
+
+@pytest.fixture
+def trained_model() -> Dict:
```

Comment:
I'm kinda tempted to say we shouldn't have stuff in conftest and we should avoid fixtures for the prototype. Just so everything is clear and explicit.. WDYT?